src/main/java/seedu/moneygowhere/ui/StatsPanel.java

```diff
@@ -20,20 +21,22 @@
     @FXML
     private StackPane panePlaceholder;
 
-    public StatsPanel(LinkedHashMap<String, Double> statsData, String commandResult) {
+    public StatsPanel(LinkedHashMap<String, Double> statsData, String commandResult, Currency currencyInUse) {
         super(FXML);
-        loadData(statsData, commandResult);
+        loadData(statsData, commandResult, currencyInUse);
     }
 
+    //@@author choongyx
     /**
      * Constructs the pie chart with the data.
      */
-    private void loadData(LinkedHashMap<String, Double> statsData, String commandResult) {
+    private void loadData(LinkedHashMap<String, Double> statsData, String commandResult, Currency currency) {
         ObservableList<PieChart.Data> pieChartData = FXCollections.observableArrayList();
 
         for (Map.Entry<String, Double> i : statsData.entrySet()) {
-            pieChartData.add(new PieChart.Data(i.getKey() + String.format(" ($%.2f)", i.getValue()),
-                    Math.round(i.getValue())));
+            double newCost = currency.rate * i.getValue();
+            pieChartData.add(new PieChart.Data(i.getKey() + String.format(" (%s%.2f)", currency.symbol, newCost),
```

Comment:
This may be refactored by putting i.getKey() into String.format and extracting it out into a new variable